test/bdd/bddtests_test.go

```diff
@@ -79,6 +81,28 @@ func runBDDTests(tags, format string) int { //nolint: gocognit
 						panic(fmt.Sprintf("Invalid value found in 'TEST_SLEEP': %s", e))
 					}
 				}
+
+				fmt.Printf("*** testSleep=%d", testSleep)
```

Comment:
it looks weird to have two testSleep in a row on command line - it would be nice to mention why for example second sleep comment could also have additional comment: testSleep=15 starting discovery server after generating configuration.

Reply:
For the first one you can say something like "waiting for the system to start ..." or similar (your English is much better).

Reply:
Added the additional note to the second testSleep message.